client/src/plus/collections/collection.tsx

```diff
@@ -110,10 +111,72 @@ export default function CollectionComponent() {
   );
 }
 
+function FrequentlyViewedCollectionComponent() {
+  let [size, setSize] = useState(0);
+  let [atEnd, setAtEnd] = useState(false);
+  let frequentlyViewed = useFrequentlyViewed(10, size, setAtEnd);
+  let _vals: Item[] | undefined = frequentlyViewed?.items.map((v) => {
+    return {
+      collection_id: frequentlyViewed?.name || "",
+      updated_at: dayjs(v.timestamps[0]).toISOString(),
+      created_at: dayjs(v.timestamps[0]).toISOString(),
+      notes: "",
+      id: v.serial.toString(),
+      parents: v.parents || [],
+      url: v.url,
+      title: v.title,
+    };
+  });
+  let mutate = (_vals) => Promise.resolve(_vals);
+  useScrollToTop();
+
+  return (
+    <div className="collections collections-collection">
+      <header>
+        <Container>
+          <Link to="../" className="exit">
+            &larr; Back
+          </Link>
+          <h1>{frequentlyViewed?.name}</h1>
```

Comment:
I have created a template empty 'Frequently viewed Collection' for the static bits . Although this will all be in hidden in the absence of any items now anyway :)